Code/GraphMol/Wrap/Mol.cpp

```diff
@@ -199,6 +200,101 @@ namespace RDKit {
     return mol.getNumAtoms(onlyExplicit);
   }
 
+  //----------------------------------------------------------------------------
+  // Atom Bookmark Interface
+  void SetAtomBookmark(ROMol *mol, Atom *atom, int mark)
+  {
+    // dangerous as there is not concept of shared pointers here (or internal
+    //  to the underlying function.  I.e. the atom can be deleted
+    //  EVEN if stored as a bookmark
+    mol->setAtomBookmark(atom, mark);
+  }
+
+  void ReplaceAtomBookmark(ROMol *mol, Atom *atom, int mark)
+  {
+    // dangerous as there is not concept of shared pointers here (or internal
+    //  to the underlying function.  I.e. the atom can be deleted
+    //  EVEN if stored as a bookmark
+    mol->replaceAtomBookmark(atom, mark);
+  }
+
+  typedef std::vector<Atom*> AtomVector;
+  AtomVector *GetAllAtomsWithBookmark(ROMol *mol, int mark)
```

Comment:
It was laziness mainly because I didn't know how to convert the Atom* to a python object. 

I have since wrapped the std::list directly through boost which is a cheaper operation.

https://github.com/bp-kelley/rdkit/commit/7033f066f4988ad76cc555c73426c6f71cb7e151